### PR TITLE
Scrub and Refactor

### PR DESCRIPTION
**Pull request by jofultz**
## Type of PR

- [ ] Documentation changes
- [X] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## Purpose of PR
This contains various refactoring:
- data access base class added
- several literals moved to constants
- named KV methods that are synchronous to suffix with "Sync"
- use of prefetch KV name-value pairs
- removal of unused imports 

## Issues Closed or Referenced

- References #65

### CODE REVIEW

File: src/main/java/com/microsoft/cse/helium/app/config/CosmosDbConfig.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        _cosmosContainer = _configService.getConfigEntries().get(Constants.COSMOS_COLLECTION_KEYNAME);
+        _cosmosDatabase = _configService.getConfigEntries().get(Constants.COSMOS_DATABASE_KEYNAME);
+
+        _requestOptions.setConsistencyLevel(ConsistencyLevel.CONSISTENT_PREFIX);
+        _requestOptions.setScriptLoggingEnabled(true);
+    }
+
+    @Bean
+    @Primary
+    public CosmosDBConfig BuildCosmosDBConfig() {
```
Comment (jyotsnaravikumar) [inline src/main/java/com/microsoft/cse/helium/app/config/CosmosDbConfig.java]: we can follow standard camel case nomenclature buildCosmosDBConfig

File: src/main/java/com/microsoft/cse/helium/app/controllers/SecretController.java [hunk 2]
```diff
@@ -17,7 +17,7 @@
 import reactor.core.publisher.Mono;
 
 import com.microsoft.cse.helium.app.services.keyvault.IKeyVaultService;
-import com.microsoft.cse.helium.app.services.keyvault.KeyVaultService;
+//import com.microsoft.cse.helium.app.services.keyvault.KeyVaultService;
 
 @RestController
 @RequestMapping (value = "api/secret")
```
Comment (jyotsnaravikumar) [inline src/main/java/com/microsoft/cse/helium/app/controllers/SecretController.java]: Do we need API to retrieve secrets?

Comment (jofultz) [reply src/main/java/com/microsoft/cse/helium/app/controllers/SecretController.java]: Yes, it is for MIKV.

File: src/main/java/com/microsoft/cse/helium/app/config/CosmosDbConfig.java (hunk 1 above)
Comment (jofultz) [reply src/main/java/com/microsoft/cse/helium/app/config/CosmosDbConfig.java]: Done.

**Pull request merged**